source/deployment-options/docker/upgrading-wazuh-docker.rst

```diff
@@ -23,6 +31,85 @@ This section describes how to upgrade Wazuh Docker deployments starting from ver
 
          # git checkout v|WAZUH_CURRENT_DOCKER|
 
+#. Before starting the new version, we must copy all the files that have been persisted in the docker volumes and belong to the Wazuh manager installation. This process will overwrite the files belonging to the Wazuh manager installation, but it will not delete your own customization files.
```

Comment:
```suggestion
#. Before starting the new version, we must copy all the files that have persisted in the docker volumes and belong to the Wazuh manager installation. This process will overwrite the files belonging to the Wazuh manager installation, but it will not delete your own customization files.
```